src/users/api/UserDto.ts

```diff
@@ -10,21 +11,29 @@ export class UserDto {
   @ApiProperty()
   lastName: string;
 
-  @ApiHideProperty()
+  @Exclude()
   isAdmin?: boolean;
 
-  @ApiHideProperty()
```

Comment:
Leave `@ApiHideProperty()` for  excluded fields

Reply:
+